gputop/remote/gputop.js

```diff
@@ -831,7 +855,7 @@ Gputop.prototype.load_emscripten = function() {
                              '//# sourceURL=gputop-web.js\n').appendTo(document.body);
                      gputop.request_features();
                      gputop.native_js_loaded_ = true;
-                     console.log("GPUTop Emscripten code loaded\n");
+                     ("GPUTop Emscripten code loaded\n");
```

Comment:
Nope, definitely not intentional, I probably removed it by accident when I was cleaning up the code.